test/sign_corrections_tests.cpp

```diff
@@ -43,14 +45,14 @@ TEST(kuka_kr6, forward_kinematics)  // NOLINT
 {
   const auto kuka = opw_kinematics::makeKukaKR6_R700_sixx<float>();
 
-  std::vector<float> joint_values = { 0.2f, 0.2f, 0.2f, 0.2f, 0.2f, 0.2f };
-  Transform<float> forward_pose = opw_kinematics::forward(kuka, &joint_values[0]);
+  Solution<float> joint_values = { 0.2f, 0.2f, 0.2f, 0.2f, 0.2f, 0.2f };
```

Comment:
This seems strange: type-wise, a `Solution` is a vector of joint angles, but semantically, only those returned by the IK solver would be considered a solution. Other vectors of joint angles are just that, vectors of joint angles.

As it is now, you're passing a solution to the FK solver, which then as a solution, returns a transform.

It might be nicer to use different types here.

Reply:
I agree. Maybe we only need to define 'Solutions` as `std::array<std::array<T, 6>, 8>` and just use `std::array<T, 6>` everywhere else for both individual solutions and joint value containers

Reply:
What about `State` or `JointState`?

Reply:
@gavanderhoorn and @marip8, I removed the typdef `Solution` and just used `std::array<T, 6>` directly.